configure.ac

```diff
@@ -6,7 +6,7 @@ AM_INIT_AUTOMAKE([1.11 subdir-objects])
 AM_SILENT_RULES([yes])
 AM_MAINTAINER_MODE([disable])
 
-VERSION_INFO="4:0:0"
+VERSION_INFO="5:0:0"
```

Comment:
Does this actually need a soname bump? Adding fields to the end of FFMS_FrameInfo should be ABI-compatible.